### PR TITLE
Include links to whitepaper and dashboard in README

### PR DESCRIPTION
**Pull request by dsargent**
Rendered version [here](https://github.com/leaf-ai/covid-xprize/blob/add-links/README.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,4 +124,11 @@ For more information and support, refer to the competition guidelines which you
 for the contest, or post your questions in the support Slack channel, to which you should have also gained access 
 upon registering.
 
+An example dashboard using Cognizant-trained predictors and prescriptors can be found 
+[here](https://evolution.ml/demos/npidashboard/). Using this dashboard you can select among different prescriptors
+from the Pareto Front to see the effect on prescriptions for intervention plans in various regions.
+
+For more background information please see also the whitepaper
```
Comment (ofrancon) [inline README.md:131]: Is it a `whitepaper` or a `research paper`?

Comment (EKMeyerson) [reply README.md:131]: `paper` or `research paper` work for me.

Comment (dsargent) [reply README.md]: Changed

**Review by ofrancon — APPROVED** [review]
lgtm

**Pull request merged**